### PR TITLE
Improvements and completeness of the ContentMetadataTags component

### PR DESCRIPTION
**Pull request by sneridagh**
Kudos to @ericof

### CODE REVIEW

File: src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx [hunk 1]
```diff
@@ -116,6 +94,20 @@ const ContentMetadataTags = (props) => {
           />
         )}
         <meta name="twitter:card" content="summary_large_image" />
+        <meta
+          property="twitter:url"
+          content={seo_canonical_url || toPublicURL(props.content['@id'])}
+        />
+        <meta property="twitter:site" content={'@ericof'} />
```
Comment (ericof) [inline src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx:101]: I would love to get the credit, but this needs to be configured somewhere else :-D

**Review by ericof — APPROVED** [review]
Please replace the hardcoded reference to my X username

File: src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx (hunk 1 above)
Comment (sneridagh) [reply src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx:101]: LOL!

Comment (sneridagh) [reply src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx:101]: @ericof We don't have that data, right?

File: src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx [hunk 2]
```diff
@@ -63,35 +54,16 @@ const ContentMetadataTags = (props) => {
...
-    const siteRootTitle = site?.['plone.site_title'];
-    const titlePart = navRootTitle || siteRootTitle;
-
-    if (includeSiteTitle && titlePart && titlePart !== title) {
-      return seo_title || `${title} ${titleAndSiteTitleSeparator} ${titlePart}`;
-    } else {
-      return seo_title || title;
-    }
-  };
-
```
Comment (erral) [inline src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx:81]: Why have you removed this?

This information isn't coming form the Header (https://github.com/plone/volto/blob/7b85e97c5e7c5046f1fce432bbbfd3f6842bd15f/src/components/theme/View/View.jsx#L238) so suddenly this feature has dissapeared

**Comment by sneridagh** [comment]
You are correct, I don't know how I overlooked completely the diff...

**Comment by sneridagh** [comment]
@erral  https://github.com/plone/volto/releases/tag/17.6.1
 https://github.com/plone/volto/releases/tag/18.0.0-alpha.3

**Pull request merged**